config/foreman-proxy-content.migrations/161216113043-move_puppet_to_top_level.rb

```diff
@@ -0,0 +1,20 @@
+answers['foreman_proxy_content'].delete('puppet')
+
+answers['puppet'] ||= {
+  'server' => true,
```

Comment:
I suppose so, although if someone actually did set that to false a lot of things would break.